integration-cli/docker_api_containers_test.go

```diff
@@ -1636,3 +1636,26 @@ func (s *DockerSuite) TestPostContainerStop(c *check.C) {
 		c.Fatal(err)
 	}
 }
+
+// #14170
+func (s *DockerSuite) TestPostContainersCreateWithStringOrSliceEntrypoint(c *check.C) {
+	config := struct {
+		Image      string
+		Entrypoint string
```

Comment:
Entrypoint is defined as a slice, why are you testing it as just a string? Is this backwards compat?